hundred_x/client.py

```diff
@@ -84,6 +137,10 @@ def send_message_to_endpoint(self, endpoint: str, method: str, message: dict, au
         """
         Send a message to an endpoint.
         """
+        if not self._validate_function(
```

Comment:
@wakamex what do you think of this approach?

Reply:
nice,  I thought we'd have to paste this call in every single function. putting it here makes sense, it's a natural bottleneck.